server/internal/graphql/resolver/meshsync.go

```diff
@@ -34,7 +34,7 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider,
 			mesherydbPath := path.Join(utils.GetHome(), ".meshery/config")
 			err := os.Mkdir(path.Join(mesherydbPath, ".archive"), os.ModePerm)
 			if err != nil && os.IsNotExist(err) {
-				createFolder := models.ErrMakeDir(err, path.Join(mesherydbPath, ".archive"))
+				createFolder := ErrMakeDir(err, path.Join(mesherydbPath, ".archive"))
```

Comment:
The purpose of Meshkit errors is to add context and give more useful information along with suggested remediation. ErrMakeDir is too generic and adds no context to the error..Use ErrResync (if this doesn't already exists)

Reply:
I see, I'm getting it slowly, thanks for those comments.
Will look into it!